indico/modules/events/notes/models/notes.py

```diff
@@ -41,7 +42,8 @@ class EventNote(LinkMixin, db.Model):
     __tablename__ = 'notes'
     allowed_link_types = LinkMixin.allowed_link_types - {LinkType.category}
     unique_links = True
-    events_backref_name = 'notes'
+    events_backref_name = 'all_notes'
```

Comment:
It's the relationship containing all notes linked to something in the event (`e.note` is the note linked to the event itself)

`notes` would work here but for attachments where there's more than one folder per object we have `attachment_folders` and `all_attachment_folders` so it's more consistent to do the same for notes.